### DISCUSSION TITLE
Opening Setup window or reinstall possible  ?

### DISCUSSION THREAD

**Discussion by simflyer787** [kubilus1/autoortho#134] [Q&A] [answered]
Hello,

i have made a slight mistake while setting up autoortho. I have selected do not show setup again in the checkbox above the fly button, tho i forgot tho choose any scenery. 

Is there any way to open the setup window again, or is it possible to reinstall the whole program ?

Thanks in advance.

**Comment by kubilus1** [comment, marked as answer]
Yeah no problem.  Just delete your local .autoortho config file.  It should
be in your Users directory.  So 'C:\Users\<yourname>\.autoortho'.  That
will start you with a fresh config.

**Reply by simflyer787** [reply]
Worked perfetly. 

Thanks for the quick reply.

Have a good day
